README.rst

```diff
@@ -44,10 +44,23 @@ The easiest way to install CMinx is via ``pip``. Simply run:
 
    pip install CMinx
 
-More detailed instructions, including how to install CMinx as part of a CMake
-build system, can be found 
+
+CMinx can also easily be used as part of a CMake workflow by adding:
+
+.. code-block:: cmake
+
+   include(FetchContent)
+   FetchContent_Declare(
+       cminx
+       GIT_REPOSITORY https://github.com/cmakepp/cminx.git
+   )
+   FetchContent_MakeAvailable(cminx)
+   cminx_gen_rst("${input_dir}" "${output_dir}")
```

Comment:
Has this example been tested? I think it should work but all of our code that I know has an additional `include(cminx)`  after the MakeAvailable() call

Reply:
I tested it locally and it worked. It would be great to check it via CI, but that's probably going to require us to have multiple deployments, which I don't want to get into right now.

As for why it works, the CMinxConfig.cmake file adds cminx_gen_rst to the current scope. The config file is loaded at the bottom of the CMakeLists txt. Finally CMakeLists.txt is loaded by FetchContent